dependencies.yaml

```diff
@@ -70,8 +87,19 @@ files:
       - test_python
 channels:
   - rapidsai
+  - rapidsai-nightly
   - conda-forge
 dependencies:
+  rapids_build_skbuild:
+    common:
+      - output_types: conda
+        packages:
+          - rapids-build-backend ~=0.3.0
```

Comment:
You upper bounded in the pyproject section but not here. We should do so consistently (CC @KyleFromNVIDIA)

Reply:
Does conda even know how to use `~=`? I thought only pip used that syntax.

Reply:
In recipes, but not in environment files. The recipe syntax actually just gets converted to comparison operators in the final package metadata

Reply:
> Does conda even know how to use `~=`

Yes: https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications.

But anyway I think this was being too cute for no real benefit. I'll change the conda ones to `>=x.x.x,<x.x.x`, matching the `pyproject` and `requirements` ones.

Reply:
changed these all to `>=0.3.0,<0.4.0.dev0` in https://github.com/rapidsai/rmm/pull/1502/commits/7aeba946df9a5c50f513e9516afb33c27a5c0435

I'll do the same in https://github.com/rapidsai/cudf/pull/15245